app/lib/efile/nj/nj1040_calculator.rb

```diff
@@ -173,6 +187,30 @@ def calculate_line_9
         number_of_line_9_exemptions * 6_000
       end
 
+      def line_59_primary
+        primary_excess = get_personal_excess(@intake.primary.ssn, :box14_ui_wf_swf, EXCESS_UI_WF_SWF_MAX)
+        primary_excess += get_personal_excess(@intake.primary.ssn, :box14_ui_hc_wd, EXCESS_UI_WF_SWF_MAX)
+        primary_excess
+      end
```

Comment:
[dust] your choice, but this could also be

```
def line_59_primary
  get_personal_excess(@intake.primary.ssn, :box14_ui_wf_swf, EXCESS_UI_WF_SWF_MAX) +
  get_personal_excess(@intake.primary.ssn, :box14_ui_hc_wd, EXCESS_UI_WF_SWF_MAX)
end
```